linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_forward_files/oval/shared.xml

```diff
@@ -0,0 +1,34 @@
+<def-group>
+  <definition class="compliance" id="no_forward_files" version="1">
+    {{{ oval_metadata("The .forward file specifies an email address to forward the user's mail to. Any .forward files should be removed.") }}}
+    <criteria>
+      <criterion test_ref="test_accounts_users_home_forward_file_existance" negate="true" />
+    </criteria>
+  </definition>
+
+  <unix:password_object id="object_accounts_users_home_forward_file_existance_objects" version="1">
+    <unix:username datatype="string" operation="pattern match">.*</unix:username>
+    <filter action="include">state_accounts_users_home_forward_file_existance_interactive_gids</filter>
+  </unix:password_object>
+
+  <unix:password_state id="state_accounts_users_home_forward_file_existance_interactive_gids" version="1">
+    <unix:user_id datatype="int" operation="greater than or equal">{{{ gid_min }}}</unix:user_id>
+  </unix:password_state>
+
+  <local_variable id="var_accounts_users_home_forward_file_existance_dirs" datatype="string" version="1"
+                  comment="Variable including all home dirs from interactive users">
+    <object_component item_field="home_dir"
+                      object_ref="object_accounts_users_home_forward_file_existance_objects"/>
+  </local_variable>
+
+  <unix:file_object id="object_accounts_users_home_forward_file_existance" version="1">
+    <unix:path var_ref="var_accounts_users_home_forward_file_existance_dirs" var_check="at least one"/>
+    <unix:filename operation="pattern match">\.forward</unix:filename>
+  </unix:file_object>
+
+  <unix:file_test id="test_accounts_users_home_forward_file_existance" check="all"
+                  check_existence="at_least_one_exists" version="1"
+                  comment=".netrc files are not group or world accessible">
```

Comment:
```suggestion
                  comment=".forward files are not group or world accessible">
```

Reply:
:man_facepalming: fixed in [478f10d](https://github.com/ComplianceAsCode/content/pull/9990/commits/478f10d91237b340025fe3ca5be521f0347e85d2)